src/gluonts/evaluation/_base.py

```diff
@@ -386,6 +386,7 @@ def get_base_metrics(
             "MASE": mase(pred_target, median_fcst, seasonal_error),
             "MAPE": mape(pred_target, median_fcst),
             "sMAPE": smape(pred_target, median_fcst),
+            "num_evaluations": pred_target.count(),
```

Comment:
I agree. I updated the code so that now we have the metric called `num_masked_target_values`, and defined it based on the case if the provided target is a masked array or not.